Projekty/Projekt1/Paulina_Jaszczuk_Adam_Frej_Jakub_Foltyn/Final_testing/WUM_projekt_klasyfikacja.ipynb

```diff
@@ -0,0 +1 @@
yqxRLcjRURETtAzP+zH2t35PDpxBY0bxNEoyX/sjdZ8DoFiWP6xMwH4G8AvV0N8srsFS0xQCBMRETlBA9o2IqewjPySAA9e1AOfN4wbTV1HwSXPQajc6R/2xUPOE5P+Y/QpkzpDIUxEROQEWWt55ss1pDeI48wuTcLbKCEV+t/gzK+fBqX74H8eA496CtUXOtMiIiInaHN2EYu35pFdWMbwJ7/CWluzHcx8Dho0gV5XuVOgxCSFMBERkRPUJj2JF28YgNdjGNw+HWNM+BvvWg5rP4fBt4A/wb0iJeYohImIiJwgYwzrswoJhix3ndOlZhvPeh58iTBojDvFScxSCBMRETlBZYEQj09eCUCHjAbhb5i/E5a8C/2uh6R0l6qTWKUQJiIiUovmbMgOv0/Ydy9CsBxOvc3doiQmKYSJiIicoDifhzbpztASt775PX+ctIKcwrLqNyotgLmvQPeLIb1DBKqUWKMQJiIiUgveHDOEoR2cW4ovfbuBfo98ztyN2UffYP4bUJILw+6IUIUSaxTCREREakGbxkm88ZPB/OfmIQeWXfnCrKrfJRksh5nPOiPkLx4HE+91po3TI1ixRJtCmIiISC2J93kZ1imDz+8+48CyM576iqVZQRZvzT3YMBSAxHRnSIplH8Lid2Huy8681BsKYSIiIrWsc2YKUyqCWF5xOX+eV8Ilz81g3Z4Cp4E/EW6bCb9e70xDf+YsH6hhKuoThTAREREXdMlM4Yt7zmDc2KEk+6FLZjLtG1cxfEXJPpjzAnS7CDJ7RL5QiRqFMBEREZd0aprC9txiCsrhhqFt8XiqGEl/3qtQkgfD74l8gRJVCmEiIiIumr42C4Dff7yMpz5bSShUaQyx8mJnxPwOI6HlgChVKNGiECYiIuKiP/6gF53SnF+3z3+1joWVO+jP/zcU7oYzfhml6iSaFMJERERclOD38sDQRM7u1hSAtbsrOueXl8CMv0HrodD2tChWKNGiECYiIhIBF/dpAXDwlUbz34B922Dkb8FU0VdM6jyFMBEREZeFrOWFr9fRrnESl/VvBWVF8O1foO3p0P7MaJcnUaIQJiIi4rLZO4Ks3JnPPed1xe/1wLxXoGAXnHW/roLVYwphIiIiLioLhPhwTRk9mqdyUa/mzou7p//VeSKy7bBolydRpBAmIiLionfmbmZPseUH/Vo644R9908o2gtnPRDt0iTKfNEuQEREpC5buCUPgDkb9vLTMzrAnH86K16/8GCj5Kbws1kQnxyFCiVaFMJERERc1KNFKu/Ph2uHtHEWnPV72LvGmbehilBmwBcftRolOhTCREREXJJfUs4jny4H4PROTZyF/W842GDpBxAsg5G/A68/ChVKNKlPmIiIiEt25pUcmF+wOefQlcFy+PJRaNoDel0Z4cokFiiEiYiIuKRzZsqBQVqvfnE2L36z7uDKhW9B9jrn9qTHG6UKJZoUwkRERFz016v6cHXXOAD+OGkl7e6byKMfzcdOewJaDYauo6JcoUSLQpiIiIiLfF4Pw1v6uOPszvRrk+YsnPsyJn87N20dxc59pdEtUKJGIUxERMRlyXGGe87twoe3nca0n/fi3oQJTAv24auSrgz901TmH95fTOoFPR0pIiISQe2WPAuhIjIuexLe2QvAZf+YeWD9f346hKHtG2MMGL3SqE5TCBMREYmU3Sth7isw4EZ69hvKhr6WiUt2sGpnPs9+uRaAa1+aA0CfVg35+PbTo1mtuEy3I0VERCJlygMQl+yMC4Zzpeui3i2497yubHz8Ql69cSB3nt0ZgEVb83hm6ppoVisu05UwERGRSFjzBaz9HM57FBpkVNnkrG6ZnNUtk8LSAC9P38DcjdkEQxavR7cl6yKFMBEREZeZUBCm3A++BEhMhyXjnRX+JOjyP0eME3bveV35ctVuZq/fS2FZgNQEjaZfFymEiYiIuCyuLBuy1oANwse3Hbrypv9C21MPWbRrXwnr9xQCEAzaSJUpEaYQJiIi4rLShCZwz3IozXcWlBfBG5dAw9bQevAR7QtKAwfm7/tgMS9cP0BPStZB6pgvIiISCSnNIKOzM62cCCW5cMGTVb6yqGfLhjz2g54AfLZsF8GQrobVRQphIiIikZSzEab/DXpeAW2HHbVZzxYNAbjj7M74vPp1XRfprIqIiETSZ/eDxwfnPVJts6c+WwXAuj0F/O2L1YR0NazOUQgTERGJlHVfwspP4dSfQ0IalBU5Uyh4RNPkeB+ZqfFMWrKDl75ZT9AqhNU16pgvIiISKTOecT6/edKZ9svsCT+bcUjTF24YwKItuYx+fgaFZUFyCstompoQwWLFbQphIiIikTLit9BhxMHvuZtg3qvQuFOVzUOVrn5d9c9Z9G2dRkl5iNM7Z3D90Lbu1iquUwgTERGJlDZDnAnAWvjPVc5rjM7/U5XN+7VpxNNX9eFX4xdjgQVbctm0t4j/LtvJVyt30z6jAfeN6qaO+ycphTAREZFoWDkR1kyB8x6D1BZHbXZZ/1Zc1r/Vge9LtznvlJyyfBcAvzq/q36Zn6QUnUVERCKtrBAm/waangJDbqnRpj1bNuSWMzsCcPc5XYj3HTnOmJwcFJ5FREQi7esnYd9WuPxl8NbsvZDWWh6fvIKM5HhuHt7epQIlEnQlTEREJJJ2r4RZz0Hf6454Z2Q4pq7YzdyNOdx1TmcaxOtayslMIUxERCRSQiGY8AuIT4Fz/lDjzQPBEE/8dyVpSX66NkthW26xC0VKpChCi4iIRMqCf8HW7yCjC3z++4PLjQeG3ArNe1e7+cqd+azZXQDAlS/MIs7rYfVjo9ysWFykECYiIhIpHh+ktYFACWyqGJy1KBvKCqD14GOGsFNapDL+1lNZv6eQX7+/mIzkOEIhi8djIlC81DaFMBERkUjpd70z7VecA88PgUbtnT5ix2CMYWC7dNZWXA3bnldCh99N4v+u68+oXs3dqlpcoj5hIiIi0TLlASjMgtHP1egpySsGtOL5a
```

Comment:
<p>Skoro z poprzednich eksperymentów wiadomo było, że dla parametru max_depth=100 wychodzą obiecujące wyniki należało umożliwić algorytmowi poszukiwanie optymalnego rozwiązania także z taką wartością tego parametru, tymczasem zdecydowano się w ramach strojenia szukać optymalnego rozwiązania dla max_depth=[3,5, 6]. Zastosowane podejście jest nielogiczne, należałoby "dostrajać" wcześniej znaleziony dobry model, a także spróbować szukać optimum w "innych regionach przestrzeni parametrów" - nie można zapomnieć o wcześniejszych wynikać i ograniczyć się do nowych poszukiwań. Mówi się, że należy zachować kompromis między explotation and exploration.</p><p> </p><p>W tym wypadku, niesłusznie wzięto do dalszych eksperymentów model, który miał gorsze parametry po strojeniu</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>